assets/app/views/directives/delete-button.html

```diff
@@ -0,0 +1,8 @@
+<div class="actions action-button pull-right">
```

Comment:
it is possible that we wouldn't always want the delete button to have pull-right, i feel like that should be up to the view that is using the delete button to decide how its placed.